data/sql/updates/pending_db_world/rev_1629290838385347720.sql

```diff
@@ -0,0 +1,56 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1629290838385347720');
+
+-- Delete Chimaera Leather
+DELETE FROM `skinning_loot_template` WHERE `item` = 15423 AND `entry` IN (7447, 7448, 7449, 8763, 8764, 10807, 11497);
```

Comment:
you need only this line to remove the item from the skinning_loot_template, don't need to re-add again all the items in any skin_loot_template, on Keira3 you can use the diff sql script and not the full one

Reply:
I already used the diff script (Keira3 3.2.0). 
Please read the full PR, I think there is a misunderstanding. 
I deleted Chimaera Leather and then changed the loot chance for all the other Skinning Loot (see above).

Reply:
you're right, sorry.